content/spin/v2/serverless-ai-api-guide.md

```diff
@@ -148,28 +148,38 @@ export const handleRequest: HandleRequest = async function (request: HttpRequest
 
 {{ startTab "Python"}}
 
-> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/spin_llm.html)
+> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/llm.html)
 
 ```python
-from spin_http import Response
-from spin_llm import llm_infer
-
-
-def handle_request(request):
-    prompt="You are a stand up comedy writer. Tell me a joke."
-    result=llm_infer("llama2-chat", prompt)
-    return Response(200,
-                    {"content-type": "text/plain"},
-                    bytes(result.text, "utf-8"))
+from spin_sdk.http import simple
+from spin_sdk.http.simple import Request, Response
+from spin_sdk import llm
+
+class IncomingHandler(simple.IncomingHandler):
+    def handle_request(self, request: Request) -> Response:
+        res = llm.infer("llama2-chat", "tell me a joke")
+        print(res.text)
+        print(res.usage)
+        res = llm.infer_with_options("llama2-chat", "what is the theory of relativity", llm.LLMInferencingParams(temperature=0.5))
+        print(res.text)
+        print(res.usage)
+        return Response(
+            200,
+            {"content-type": "text/plain"},
+            bytes("Hello from Python!", "utf-8")
+        )
 ```
 
 **General Notes**
 
-[`llm_infer` operation](https://fermyon.github.io/spin-python-sdk/spin_llm.html#spin_sdk.spin_llm.llm_infer):
+[`infer` operation](https://fermyon.github.io/spin-python-sdk/llm.html#spin_sdk.llm.infer):
 
-- It takes in a model name, prompt text, and optionally a [parameter object](https://fermyon.github.io/spin-python-sdk/spin_llm.html#spin_sdk.spin_llm.LLMInferencingParams) to control the inferencing. 
 - The model name is passed in as a string (as shown above; `"llama2-chat"`).
-- The return value is am [`InferencingResult` object](https://fermyon.github.io/spin-python-sdk/spin_llm.html#spin_sdk.spin_llm.LLMInferencingResult) - use the `text` attribute to get the inferred response.
+- The return value is am [`InferencingResult` object](https://fermyon.github.io/spin-python-sdk/wit/imports/llm.html#spin_sdk.wit.imports.llm.InferencingResult) - use the `text` attribute to get the inferred response.
+
+[`infer_with_options` operation](https://fermyon.github.io/spin-python-sdk/llm.html#spin_sdk.llm.infer_with_options):
+
+- It takes in a model name, prompt text, and optionally a [parameter object](https://fermyon.github.io/spin-python-sdk/llm.html#spin_sdk.llm.LLMInferencingParams) to control the inferencing. 
```

Comment:
`llm.LLMInferencingParams` but `llm.InferencingResult`?  Why the extra `LLM` prefix?